pax/plugins/signal_processing/BuildPeaks.py

```diff
@@ -31,9 +31,6 @@ def startup(self):
         self.first_top_ch = np.min(np.array(self.config['channels_top']))
         self.last_top_ch = np.max(np.array(self.config['channels_top']))
 
-        # tight coincidence
-        self.tight_coincidence_window = self.config.get('tight_coincidence_window', 50) // self.dt
```

Comment:
Why was this ever here? It seems it was never used?

Reply:
I was trying to add this in the classification decision during clustering, but later found out it's not necessary. But forgot to cut it away.